app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
-  before_action :authenticate_user!
   protect_from_forgery with: :exception
-
+  before_action :authenticate_with_token
+  # before_action :authenticate_user!
```

Comment:
- [ ] [OPTIONAL] Please remove commented code to make your code clean :point_down: